build.gradle

```diff
@@ -7,8 +7,8 @@ buildscript {
 
       'kotlin': '1.2.31',
 
-      'androidPlugin': '3.1.0',
-      'androidTools': '26.1.0',
+      'androidPlugin': '3.2.0-alpha10',
+      'androidTools': '26.2.0-alpha10',
```

Comment:
Why bump this now? It means we can't release until 3.2 stable which probably isn't until like June or July.

Reply:
No reason.  Just updating to the latest and greatest, similar to 3.1 alphas.  Can downgrade to 3.1.1.